addon/ajax-request.js

```diff
@@ -160,6 +160,15 @@ export default class AjaxRequest {
   }
 
   /**
+   * calls `request()` but forces `options.type` to `DELETE`
+   * alias for `del()`
+   * @public
+   */
+  delete(...args) {
+    return this.del(...args);
```

Comment:
The code above basically does the same thing.

I can change it if you really want to, but it won't much of a difference.

```js
# try in FF console
let a = function(...args) { console.log(...args); args.forEach(function(a) { console.log(`a: ${a}`); }); };
a('bat', 'bird', 'badger');

let b = function(...arguments) { console.log(...arguments); arguments.forEach(function(a) { console.log(`b: ${a}`); }); };
b('bat', 'bird', 'badger');
```